EpikCord/__init__.py

```diff
@@ -1601,6 +1601,12 @@ def __init__(self, client, data: dict):
         self.bitrate: int = data.get("bitrate")
         self.user_limit: int = data.get("user_limit")
         self.rtc_region: str = data.get("rtc_region")
+        self.__voice_client = VoiceWebsocketClient(client)
```

Comment:
How about not making that attribute private ? Just a suggestion

Reply:
well if i made it public,users have to do voice.connect(guild_id, channel_id) which we dont want.
What im saying is that users are not supposed to use this class, it is for both voice and stage channels